assets/constants.ts

```diff
@@ -6035,7 +6035,7 @@ export default {
   // Interaction IDs for which answer details cannot be solicited.
   "INTERACTION_IDS_WITHOUT_ANSWER_DETAILS": ["EndExploration", "Continue"],
 
-  "WHITELISTED_COLLECTION_IDS_FOR_SAVING_GUEST_PROGRESS": [],
+  "ALLOWLISTED_COLLECTION_IDS_FOR_SAVING_GUEST_PROGRESS": [],
```

Comment:
Wouldn't it be possible to just call this `ALLOWED_COLLECTION_IDS_FOR_SAVING_GUEST_PROGRESS`

Reply:
Yes, I changed it to `ALLOWED_COLLECTION_IDS_FOR_SAVING_GUEST_PROGRESS `